test/browsercontext.spec.js

```diff
@@ -547,12 +551,12 @@ module.exports.describe = function({testRunner, expect, playwright, CHROMIUM, FF
     });
     it('should report initialized pages', async({browser, server}) => {
       const context = await browser.newContext();
-      const pagePromise = context.waitForEvent('page').then(e => e.page());
+      const pagePromise = context.waitForEvent('page');
```

Comment:
Let's add a test for `page.waitForLoadState({waitUntil:'load'})` on a `context.newPage()`, about:blank popup and network url popup. We lost the coverage by removing `e.page()` calls that were waiting for load.